### PR TITLE
AUI-2116 Source Format

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-layout/js/aui-layout-builder-resize-col.js [hunk 1]
```diff
@@ -280,26 +280,6 @@ A.LayoutBuilderResizeCol.prototype = {
...
-     * @method _canDrag
-     * @param {Node} dragNode
-     * @protected
-     */
-    _canDrag: function(dragNode) {
-        var index;
-
-        for (index = 0; index < BREAKPOINTS.length; index++) {
-            if (BREAKPOINTS[index] !== dragNode.getData('layout-position') &&
-                this._canDrop(dragNode, BREAKPOINTS[index])) {
```
Comment (fernandosouza) [inline src/aui-layout/js/aui-layout-builder-resize-col.js]: @thiago-rocha. please, double check the necessity of this method. I couldn't understand the logic inside of the method _canDrop to evaluate this condition.

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by thiago-rocha** [comment]
Pull request submitted to https://github.com/mairatma/alloy-ui/pull/336. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**